app/Providers/EventServiceProvider.php

```diff
@@ -3,7 +3,9 @@
 namespace App\Providers;
 
 use Illuminate\Auth\Events\Registered;
+use App\Events\OfferPriceChanged;
```

Comment:
Wydaje mi się że warto to od razu wydzielić po modelach np Offer\fferPriceChanged